data_sample/tests/test_login_example.py

```diff
@@ -0,0 +1,17 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Login"""
+
+
+class LoginTestCase():
```

Comment:
Since this is showing an example of a class which probably will be a unittest TestCase child is better to show it here. Otherwise a plain test function (test_loging_1) can be used.